src/components/Plugins/PluginRegisterSystem.tsx

```diff
@@ -64,7 +64,7 @@ const PluginRegisterSystem: React.FC<{ baseurl?: string }> = ({ baseurl }) => {
     <HidingElement>
       {toRegister.map((plg) => (
         <iframe
-          key={plg.name}
+          key={plg.identifier}
```

Comment:
is there a chance of having duplicate named plugins?  is that why we're removing `name` ?

Reply:
Yes. In theory duplicate names should work. I believe identifier comes from actual folder structure so there should not be duplicates